docs/aci-compose-features.md

```diff
@@ -0,0 +1,118 @@
+| service.deploy.restart_policy  | ✓  |  One of: any - none - on-failure. Restriction: All services must have the ame restart policy. The entire ACI container group will be restarted if needed. | 
+| service.deploy.labels          | x  |  ACI does not have container-level labels. In addition, Azure tag are too restrictive to support Docker container labels.| 
+| service.devices                | x  | 
+| service.depends_on             | x  | 
+| service.dns                    | x  | 
+| service.dns_search             | x  | 
+| service.domainname             | ✓  |  Mapped to ACI DNSLabelName. Restriction: all services must specify the same domainname, if specified.
+| service.tmpfs                  | x  |  
+| service.entrypoint             | x  |  ACI allows to override Command, but not Entrypoint. 
+| service.env_file               | ✓  |  
+| service.environment            | ✓  |  
+| service.expose                 | x  |   
+| service.extends                | x  |  
+| service.external_links         | x  |  
+| service.extra_hosts            | x  |  
+| service.group_add              | x  |  
+| service.healthcheck            | x  |  
+| service.hostname               | x  |  
+| service.image                  | ✓  |  Private images will be accessible if the user is logged in against the corresponding registry at deploy time. Users will be automatically logged in to Azure Container Registry from Azure login if possible.
+| service.isolation              | x  |  
+| service.labels                 | x  |  ACI does not have container-level labels. In addition, Azure tag are too restrictive to support Docker container labels.  |
+| service.links                  | x  |  
+| service.logging                | x  |  
+| service.network_mode           | x  |  
+| service.networks               | x  |  Communication between services is supported by defining mapping for each service in `/etc/hosts` in the container group. Each service can resolve names for other services and the resulting network calls will be redirected to `localhost`.
+| service.pid                    | x  |  
+| service.ports                  | ✓  |  
+| service.secrets                | ✓  |    
+| service.security_opt           | x  |  
+| service.stop_grace_period      | x  |  
+| service.stop_signal            | x  |  
+| service.sysctls                | x  |  
+| service.ulimits                | x  |  
+| service.userns_mode            | x  |  
+| service.volumes                | ✓  |  Mapped to AZure File Shares. See #persistent_volumes
+| service.restart                | x  |  Replaced by service.deployment.restrt_policy                |
+|                                |    |                                                              |
+| __Volume__                     | x  |                                                              |
+| driver                         | ✓  |                                                              |
```

Comment:
Suggestion: specify what the Azure File Share valid driver name and options are

Reply:
Oh, I see, there is more info below. How to add a comment with a link to the relevant paragraph?